lib/sdr/services_api.rb

```diff
@@ -20,15 +21,15 @@ module Sdr
   # @see https://github.com/sul-dlss/druid-tools
   #
   class ServicesAPI < Sdr::Base
+    extend Deprecation
+    self.deprecation_horizon = 'sdr-services-app is being replaced by preservation-catalog'
 
     helpers do
 
       def latest_version
-        unless @latest_version
-          @latest_version = Stanford::StorageServices.current_version(params[:druid])
-        end
-        @latest_version
+        @latest_version ||= Stanford::StorageServices.current_version(params[:druid])
```

Comment:
Consider adding a Honeybadger notification here so that we know when it's truly dead.

Reply:
I just pushed a commit that does this.

I don't know enough about Sinatra to know if I need it in both places ... but it SEEMS so ...